impeller/tools/impeller.gni

```diff
@@ -18,7 +18,7 @@ declare_args() {
   impeller_enable_opengles = is_mac || is_linux || is_win || is_android
 
   # Whether the Vulkan backend is enabled.
-  impeller_enable_vulkan = is_linux || is_android
+  impeller_enable_vulkan = is_mac || is_linux || is_android
```

Comment:
Is the Vulkan playground harness configured to use the SwiftShader ICD by default on MacOS? If not, might be worth adding a doc about setting up Vulkan for MacOS [here](https://github.com/flutter/engine/tree/main/impeller/docs) (getting LunarG SDK, environment vars for MoltenVK, etc). FYI @chinmaygarde

Reply:
added a issue https://github.com/flutter/flutter/issues/119083